packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -1143,6 +1143,9 @@ const fn = new lambda.Function(this, 'Lambda_with_IPv6_VPC', {
 });
 ```
 
+To allow outbound IPv6 traffic by default, set `allowAllIpv6Outbound` to true.
```

Comment:
What about defining a `## Outbound traffic` section that explains the behavior for both `allowAllOutbound` and `allowAllIpv6Outbound`?

Reply:
Updated. Thanks for the suggestion.